### DISCUSSION TITLE
Styled-System Folders

### DISCUSSION THREAD

**Discussion by cgatian** [chakra-ui/panda#2966] [Q&A]
In a monorepo, Is it expected for each `panda.config.ts` file to generate a `styled-system` folder per each app?  I was assuming you could build your preset in one library and export the `styled-system` folder. Then apps would consume this via package.

Every now and then I get a `styled-system` folder generated in my app folders, nothing is referencing it, so I just remove it.

**Comment by astahmer** [comment]
if you use the `panda` command it will generate both the styled-system AND the css
whereas if you use `panda cssgen` it will only generate the css
(and if you use `panda codegen` it will only generate the css)

--- 

or if you're using the postcss plugin you can add this

```ts
import { defineConfig, definePlugin } from '@pandacss/dev';

const skipCodegenPlugin = definePlugin({
    name: 'skip-codegen',
    hooks: {
        'codegen:prepare': () => {
            return [];
        },
    },
});

export default defineConfig({
    plugins: [skipCodegenPlugin],
    // ...
```

**Reply by cgatian** [reply]
@astahmer, thank you for the response. It must be something funky in my setup. I'm seeing `styled-system` folders get created in packages that don't use the `panda` command. 

I'll try to track this down. I just wanted to make sure it was _unexpected_ for an app dependant on a styled system package, to also create it's own `styled-system` folder.
